tests/test_association.py

```diff
@@ -55,53 +55,43 @@
               'filter': 'r'}
 
 
-def createTestPoints(pointLocsDeg,
+def createTestPoints(nPoints,
                      startId=0,
-                     schema=None,
-                     scatterArcsec=1.0,
-                     indexerIds=None,
-                     associatedIds=None):
+                     schema=None):
     """Create dummy DIASources or DIAObjects for use in our tests.
 
     Parameters
     ----------
-    pointLocsDeg : array-like (N, 2) of `float`s
-        Positions of the test points to create in RA, DEC.
+    nPoints : `int`
+        Number of data points to create.
     startId : `int`
         Unique id of the first object to create. The remaining sources are
         incremented by one from the first id.
     schema : `lsst.afw.table.Schema`
         Schema of the objects to create. Defaults to the DIASource schema.
-    scatterArcsec : `float`
-        Scatter to add to the position of each DIASource.
-    indexerIds : `list` of `ints`s
-        Id numbers of pixelization indexer to store. Must be the same length
-        as the first dimension of point_locs_deg.
-    associatedIds : `list` of `ints`s
-        Id numbers of associated DIAObjects to store. Must be the same length
-        as the first dimension of point_locs_deg.
 
     Returns
     -------
     testPoints : `lsst.afw.table.SourceCatalog`
         Catalog of points to test.
     """
     if schema is None:
-        schema = make_minimal_dia_source_schema()
+        schema = make_dia_source_schema()
     sources = afwTable.SourceCatalog(schema)
 
-    for src_idx, (ra, dec,) in enumerate(pointLocsDeg):
+    for src_idx in range(nPoints):
         src = sources.addNew()
+        # Set everything to a simple default value.
+        for subSchema in schema:
+            if subSchema.getField().getTypeString() == "Angle":
+                continue
+            elif subSchema.getField().getTypeString() == "String":
+                src[subSchema.getField().getName()] = 'g'
```

Comment:
It took me a while to figure out that you're assuming this is a filter field; might want to explicitly say that.